--- Makefile.org	2025-07-30 18:54:18.000000000 +0900
+++ Makefile	2025-10-13 18:32:32.757032105 +0900
@@ -50,7 +50,7 @@
 # <http://www.gnu.org/licenses/>.
 #
 
-
+DATE := $(shell date +%Y-%m-%d)
 
 am__is_gnu_make = { \
   if test -z '$(MAKELEVEL)'; then \
@@ -852,8 +852,8 @@
 MANCONF = -Dman
 TEXI2POD = perl $(top_srcdir)/../etc/texi2pod.pl $(AM_MAKEINFOFLAGS)
 POD2MAN = pod2man --center="GNU Development Tools" \
-	 --release="binutils-$(VERSION)" --section=1
-
+	 --release="binutils-$(VERSION)" --section=1 --language=ja
+FIXMAN = $(AWK) '{gsub(/NAME/,"名前");gsub(/SYNOPSIS/, "書式");gsub(/DESCRIPTION/, "説明");gsub(/OPTIONS/,"オプション");gsub(/SEE ALSO/,"関連項目");gsub(/COPYRIGHT/,"著作権");gsub(/xrefなので削る/,"");print} END{print ".SH 日本語訳\n対応バージョン:$(VERSION)\n翻訳日:$(DATE)\n翻訳者:$(TRANSLATOR)"}'
 
 # List of man pages generated from binutils.texi
 man_MANS = \
@@ -2037,105 +2037,105 @@
 doc/addr2line.1:	$(binutils_TEXI) doc/$(am__dirstamp)
 	$(AM_V_GEN)touch $@
 	$(AM_V_at)-$(TEXI2POD) $(MANCONF) -Daddr2line < $(binutils_TEXI) > addr2line.pod
-	$(AM_V_at)-($(POD2MAN) addr2line.pod | sed -e '/^.if n .na/d' > $@.T$$$$ && \
+	$(AM_V_at)-($(POD2MAN) addr2line.pod | sed -e '/^.if n .na/d'|$(FIXMAN) > $@.T$$$$ && \
 		mv -f $@.T$$$$ $@) || (rm -f $@.T$$$$ && exit 1)
 	$(AM_V_at)rm -f addr2line.pod
 
 doc/ar.1:	$(binutils_TEXI) doc/$(am__dirstamp)
 	$(AM_V_GEN)touch $@
 	$(AM_V_at)-$(TEXI2POD) $(MANCONF) -Dar < $(binutils_TEXI) > ar.pod
-	$(AM_V_at)-($(POD2MAN) ar.pod | sed -e '/^.if n .na/d' > $@.T$$$$ && \
+	$(AM_V_at)-($(POD2MAN) ar.pod | sed -e '/^.if n .na/d'|$(FIXMAN) > $@.T$$$$ && \
 		mv -f $@.T$$$$ $@) || (rm -f $@.T$$$$ && exit 1)
 	$(AM_V_at)rm -f ar.pod
 
 doc/dlltool.1:	$(binutils_TEXI) doc/$(am__dirstamp)
 	$(AM_V_GEN)touch $@
 	$(AM_V_at)-$(TEXI2POD) $(MANCONF) -Ddlltool < $(binutils_TEXI) > dlltool.pod
-	$(AM_V_at)-($(POD2MAN) dlltool.pod | sed -e '/^.if n .na/d' > $@.T$$$$ && \
+	$(AM_V_at)-($(POD2MAN) dlltool.pod | sed -e '/^.if n .na/d'|$(FIXMAN) > $@.T$$$$ && \
 		mv -f $@.T$$$$ $@) || (rm -f $@.T$$$$ && exit 1)
 	$(AM_V_at)rm -f dlltool.pod
 
 doc/nm.1:	$(binutils_TEXI) doc/$(am__dirstamp)
 	$(AM_V_GEN)touch $@
 	$(AM_V_at)-$(TEXI2POD) $(MANCONF) -Dnm < $(binutils_TEXI) > nm.pod
-	$(AM_V_at)-($(POD2MAN) nm.pod | sed -e '/^.if n .na/d' > $@.T$$$$ && \
+	$(AM_V_at)-($(POD2MAN) nm.pod | sed -e '/^.if n .na/d'|$(FIXMAN) > $@.T$$$$ && \
 		mv -f $@.T$$$$ $@) || (rm -f $@.T$$$$ && exit 1)
 	$(AM_V_at)rm -f nm.pod
 
 doc/objcopy.1:	$(binutils_TEXI) doc/$(am__dirstamp)
 	$(AM_V_GEN)touch $@
 	$(AM_V_at)-$(TEXI2POD) $(MANCONF) -Dobjcopy < $(binutils_TEXI) > objcopy.pod
-	$(AM_V_at)-($(POD2MAN) objcopy.pod | sed -e '/^.if n .na/d' > $@.T$$$$ && \
+	$(AM_V_at)-($(POD2MAN) objcopy.pod | sed -e '/^.if n .na/d'|$(FIXMAN) > $@.T$$$$ && \
 		mv -f $@.T$$$$ $@) || (rm -f $@.T$$$$ && exit 1)
 	$(AM_V_at)rm -f objcopy.pod
 
 doc/objdump.1:	$(binutils_TEXI) doc/$(am__dirstamp)
 	$(AM_V_GEN)touch $@
 	$(AM_V_at)-$(TEXI2POD) $(MANCONF) -Dobjdump < $(binutils_TEXI) > objdump.pod
-	$(AM_V_at)-($(POD2MAN) objdump.pod | sed -e '/^.if n .na/d' > $@.T$$$$ && \
+	$(AM_V_at)-($(POD2MAN) objdump.pod | sed -e '/^.if n .na/d'|$(FIXMAN) > $@.T$$$$ && \
 		mv -f $@.T$$$$ $@) || (rm -f $@.T$$$$ && exit 1)
 	$(AM_V_at)rm -f objdump.pod
 
 doc/ranlib.1:	$(binutils_TEXI) doc/$(am__dirstamp)
 	$(AM_V_GEN)touch $@
 	$(AM_V_at)-$(TEXI2POD) $(MANCONF) -Dranlib < $(binutils_TEXI) > ranlib.pod
-	$(AM_V_at)-($(POD2MAN) ranlib.pod | sed -e '/^.if n .na/d' > $@.T$$$$ && \
+	$(AM_V_at)-($(POD2MAN) ranlib.pod | sed -e '/^.if n .na/d'|$(FIXMAN) > $@.T$$$$ && \
 		mv -f $@.T$$$$ $@) || (rm -f $@.T$$$$ && exit 1)
 	$(AM_V_at)rm -f ranlib.pod
 
 doc/readelf.1:	$(binutils_TEXI) doc/$(am__dirstamp)
 	$(AM_V_GEN)touch $@
 	$(AM_V_at)-$(TEXI2POD) $(MANCONF) -Dreadelf < $(binutils_TEXI) > readelf.pod
-	$(AM_V_at)-($(POD2MAN) readelf.pod | sed -e '/^.if n .na/d' > $@.T$$$$ && \
+	$(AM_V_at)-($(POD2MAN) readelf.pod | sed -e '/^.if n .na/d'|$(FIXMAN) > $@.T$$$$ && \
 		mv -f $@.T$$$$ $@) || (rm -f $@.T$$$$ && exit 1)
 	$(AM_V_at)rm -f readelf.pod
 
 doc/size.1:	$(binutils_TEXI) doc/$(am__dirstamp)
 	$(AM_V_GEN)touch $@
 	$(AM_V_at)-$(TEXI2POD) $(MANCONF) -Dsize < $(binutils_TEXI) > size.pod
-	$(AM_V_at)-($(POD2MAN) size.pod | sed -e '/^.if n .na/d' > $@.T$$$$ && \
+	$(AM_V_at)-($(POD2MAN) size.pod | sed -e '/^.if n .na/d'|$(FIXMAN) > $@.T$$$$ && \
 		mv -f $@.T$$$$ $@) || (rm -f $@.T$$$$ && exit 1)
 	$(AM_V_at)rm -f size.pod
 
 doc/strings.1:	$(binutils_TEXI) doc/$(am__dirstamp)
 	$(AM_V_GEN)touch $@
 	$(AM_V_at)-$(TEXI2POD) $(MANCONF) -Dstrings < $(binutils_TEXI) > strings.pod
-	$(AM_V_at)-($(POD2MAN) strings.pod | sed -e '/^.if n .na/d' > $@.T$$$$ && \
+	$(AM_V_at)-($(POD2MAN) strings.pod | sed -e '/^.if n .na/d'|$(FIXMAN) > $@.T$$$$ && \
 		mv -f $@.T$$$$ $@) || (rm -f $@.T$$$$ && exit 1)
 	$(AM_V_at)rm -f strings.pod
 
 doc/strip.1:	$(binutils_TEXI) doc/$(am__dirstamp)
 	$(AM_V_GEN)touch $@
 	$(AM_V_at)-$(TEXI2POD) $(MANCONF) -Dstrip < $(binutils_TEXI) > strip.pod
-	$(AM_V_at)-($(POD2MAN) strip.pod | sed -e '/^.if n .na/d' > $@.T$$$$ && \
+	$(AM_V_at)-($(POD2MAN) strip.pod | sed -e '/^.if n .na/d'|$(FIXMAN) > $@.T$$$$ && \
 		mv -f $@.T$$$$ $@) || (rm -f $@.T$$$$ && exit 1)
 	$(AM_V_at)rm -f strip.pod
 
 doc/elfedit.1:	$(binutils_TEXI) doc/$(am__dirstamp)
 	$(AM_V_GEN)touch $@
 	$(AM_V_at)-$(TEXI2POD) $(MANCONF) -Delfedit < $(binutils_TEXI) > elfedit.pod
-	$(AM_V_at)-($(POD2MAN) elfedit.pod | sed -e '/^.if n .na/d' > $@.T$$$$ && \
+	$(AM_V_at)-($(POD2MAN) elfedit.pod | sed -e '/^.if n .na/d'|$(FIXMAN) > $@.T$$$$ && \
 		mv -f $@.T$$$$ $@) || (rm -f $@.T$$$$ && exit 1)
 	$(AM_V_at)rm -f elfedit.pod
 
 doc/windres.1:	$(binutils_TEXI) doc/$(am__dirstamp)
 	$(AM_V_GEN)touch $@
 	$(AM_V_at)-$(TEXI2POD) $(MANCONF) -Dwindres < $(binutils_TEXI) > windres.pod
-	$(AM_V_at)-($(POD2MAN) windres.pod | sed -e '/^.if n .na/d' > $@.T$$$$ && \
+	$(AM_V_at)-($(POD2MAN) windres.pod | sed -e '/^.if n .na/d'|$(FIXMAN) > $@.T$$$$ && \
 		mv -f $@.T$$$$ $@) || (rm -f $@.T$$$$ && exit 1)
 	$(AM_V_at)rm -f windres.pod
 
 doc/windmc.1:	$(binutils_TEXI) doc/$(am__dirstamp)
 	$(AM_V_GEN)touch $@
 	$(AM_V_at)-$(TEXI2POD) $(MANCONF) -Dwindmc < $(binutils_TEXI) > windmc.pod
-	$(AM_V_at)-($(POD2MAN) windmc.pod | sed -e '/^.if n .na/d' > $@.T$$$$ && \
+	$(AM_V_at)-($(POD2MAN) windmc.pod | sed -e '/^.if n .na/d'|$(FIXMAN) > $@.T$$$$ && \
 		mv -f $@.T$$$$ $@) || (rm -f $@.T$$$$ && exit 1)
 	$(AM_V_at)rm -f windmc.pod
 
 doc/cxxfilt.man:	$(binutils_TEXI) doc/$(am__dirstamp)
 	$(AM_V_GEN)touch $@
 	$(AM_V_at)-$(TEXI2POD) $(MANCONF) -Dcxxfilt < $(binutils_TEXI) > $(DEMANGLER_NAME).pod
-	$(AM_V_at)-($(POD2MAN) $(DEMANGLER_NAME).pod | sed -e '/^.if n .na/d' > $@.T$$$$ && \
+	$(AM_V_at)-($(POD2MAN) $(DEMANGLER_NAME).pod | sed -e '/^.if n .na/d'|$(FIXMAN) > $@.T$$$$ && \
 		mv -f $@.T$$$$ $@) || (rm -f $@.T$$$$ && exit 1)
 	$(AM_V_at)rm -f $(DEMANGLER_NAME).pod
 
